conf/modules.config

```diff
@@ -928,25 +928,25 @@ process{
 // PAIR_VARIANT_CALLING
 
     //ASCAT
-    if (params.tools && params.tools.contains('ascat')) {
-        withName: 'ASCAT' {
-            ext.args = {
-                [
-                "gender": meta.sex,
-                //"genomeVersion": "hg19"
-                //"purity": (optional),
-                //"ploidy": params.ploidy,
-                //"gc_files": (optional),
-                //"minCounts": (optional),
-                //"chrom_names": (optional),
-                //"min_base_qual": (optional),
-                //"min_map_qual": (optional),
-                //"ref_fasta": (optional),
-                //"skip_allele_counting_tumour": (optional),
-                //"skip_allele_counting_normal": (optional)
-                ]
-            }
-        }
+    withName: 'ASCAT' {
+
+        ext.args = {[
+            "gender": meta.gender,
+            "genomeVersion": params.ascat_genome,
```

Comment:
the new params should be added to the nextflow.config with their default params (if necessary) and to the schema.json. (For this you can use `nf-core schema build`to save yourself a headache when trying to annotate manually)